### PR TITLE
feature(tooling): dump size.txt data into a parseable format

### PR DESCRIPTION
**Pull request by erwinmombay**

### CODE REVIEW

**Comment by erwinmombay** [comment]
@cramforce csv data of size over time [here](https://docs.google.com/spreadsheets/d/1gGBGQj-KDkr0zlLyQY1K5l5_wjBd1Ox0-mS9EFQwcGE/edit?usp=sharing)

**Comment by erwinmombay** [comment]
will try and clean up the data some more and see if i can create the chart in google docs

**Comment by erwinmombay** [comment]
image

**Comment by erwinmombay** [comment]
@cramforce PTAL. ready for review, added tests and comments.

File: build-system/tasks/csvify-size/test.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+var test = require('ava');
```
Comment (cramforce) [inline build-system/tasks/csvify-size/test.js]: Is this test running on travis?

Comment (erwinmombay) [inline build-system/tasks/csvify-size/test.js]: nope, didnt want to slow down builds with these tests, so just running this offline when needed

Comment (cramforce) [inline build-system/tasks/csvify-size/test.js]: I think we should run them. Otherwise we will forget. I bet this runs in like 300ms or so.

Comment (erwinmombay) [inline build-system/tasks/csvify-size/test.js]: done. added config into package.json and added `npm run node-test` on .travis.yaml file

**Comment by erwinmombay** [comment]
@cramforce PTAL

**Comment by cramforce** [comment]
LGTM

**Pull request merged**